src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -2279,7 +2270,9 @@ private static object ConvertNumericThroughDouble(object valueToConvert, Type re
                     valueToConvert.ToString(), resultType.ToString(), wmiClassException.Message);
             }
         }
+#endif
 
+#if !CORECLR // No CommaDelimitedStringCollection In CoreCLR
         // System.Configuration.CommaDelimitedStringCollection is derived from the StringCollection class
         private static System.Configuration.CommaDelimitedStringCollection ConvertToCommaDelimitedStringCollection(object valueToConvert,
```

Comment:
The CommaDelimitedStringCollection class is in .Net Core 2.0+.
https://apisof.net/catalog/System.Configuration.CommaDelimitedStringCollection